compiler/src/lntoamm/opcodes.ts

```diff
@@ -29,6 +29,22 @@ opcodeScope.put(
 )
 Type.builtinTypes['Array'].solidify(['any'], opcodeScope)
 Type.builtinTypes['Array'].solidify(['anythingElse'], opcodeScope)
+Type.builtinTypes['KeyVal'].solidify(['string', 'string'], opcodeScope)
+Type.builtinTypes['Array'].solidify(['KeyVal<string, string>'], opcodeScope)
+// HTTP server opcode-related builtin Types hackery, also defined in std/http.ln
+Type.builtinTypes.InternalRequest.properties = {
+  method: opcodeScope.get('string') as Type,
+  url: opcodeScope.get('string') as Type,
+  headers: opcodeScope.get('Array<KeyVal<string, string>>') as Type,
+  body: opcodeScope.get('string') as Type,
+  connId: opcodeScope.get('int64') as Type,
+}
+Type.builtinTypes.InternalResponse.properties = {
+  status: opcodeScope.get('int64') as Type,
+  headers: opcodeScope.get('Array<KeyVal<string, string>>') as Type,
+  body: opcodeScope.get('string') as Type,
+  connId: opcodeScope.get('int64') as Type,
+}
```

Comment:
I despise this hackery here, but you can't define a `static` element of a class that is self-referential, so constructing the `builtinTypes` field on `Type` will need to be reworked completely to avoid this, I realized, because it is itself referring to a built-in Type that is generic in its definition (`Array<KeyVal<string, string>>`) and wasn't getting defined properly before.